<ui version="4.0" >
 <class>ExportGraphicsDialog</class>
 <widget class="QDialog" name="ExportGraphicsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export to Graphics File</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="TextLabel2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Save location:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>File format:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4" >
    <widget class="QLabel" native="1" name="_saveLocation" />
   </item>
   <item row="1" column="1" colspan="4" >
    <widget class="QComboBox" name="_comboBoxFormats" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5" >
    <widget class="QCheckBox" name="_saveEPSVector" >
     <property name="text" >
      <string>Save EPS in vector format</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5" >
    <widget class="QGroupBox" name="buttonGroup2" >
     <property name="title" >
      <string>Save Options</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="_saveOnce" >
        <property name="whatsThis" >
         <string>Do not autosave.  Instead, on OK or Apply, save once.</string>
        </property>
        <property name="text" >
         <string>Save Once</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_autosave" >
        <property name="toolTip" >
         <string>Save image on timer</string>
        </property>
        <property name="whatsThis" >
         <string>When checked, the image will be repeatedly saved using the indicated period.  This is useful, for example, for live web page images.</string>
        </property>
        <property name="text" >
         <string>Autosave every</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="_period" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>Autosave timer</string>
        </property>
        <property name="suffix" >
         <string>s</string>
        </property>
        <property name="maximum" >
         <number>100000</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="4" >
    <widget class="QPushButton" name="_cancel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" >
    <widget class="QPushButton" name="_Apply" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Apply</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" >
    <widget class="QPushButton" name="_ok" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Ok</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5" >
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Size</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel3" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Width:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_xSize</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="_xSize" >
          <property name="whatsThis" >
           <string>Width (in Pixels) of the image.</string>
          </property>
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>5</number>
          </property>
          <property name="singleStep" >
           <number>20</number>
          </property>
          <property name="value" >
           <number>640</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel4" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Height:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_ySize</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="_ySize" >
          <property name="whatsThis" >
           <string>Height of the image (in pixels).</string>
          </property>
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>5</number>
          </property>
          <property name="singleStep" >
           <number>20</number>
          </property>
          <property name="value" >
           <number>480</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>120</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_2" >
          <property name="text" >
           <string>Specify:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_comboBoxSizeOption" >
          <item>
           <property name="text" >
            <string>width and height</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>size of square</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>width, maintaining aspect ratio</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>height, maintaining aspect ratio</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>61</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="5" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>380</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="5" >
    <widget class="QGroupBox" name="buttonGroup1" >
     <property name="title" >
      <string>Windows</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="_radioButtonAll" >
        <property name="whatsThis" >
         <string>Plot all windows in separate files.  The window number  will be appended to the file names.</string>
        </property>
        <property name="text" >
         <string>&amp;All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_radioButtonActive" >
        <property name="whatsThis" >
         <string>Only plot the active window.</string>
        </property>
        <property name="text" >
         <string>Acti&amp;ve only</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>160</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>_saveLocation</tabstop>
  <tabstop>_comboBoxFormats</tabstop>
  <tabstop>_comboBoxSizeOption</tabstop>
  <tabstop>_xSize</tabstop>
  <tabstop>_ySize</tabstop>
  <tabstop>_radioButtonActive</tabstop>
  <tabstop>_saveOnce</tabstop>
  <tabstop>_period</tabstop>
  <tabstop>_ok</tabstop>
  <tabstop>_Apply</tabstop>
  <tabstop>_cancel</tabstop>
  <tabstop>_radioButtonAll</tabstop>
  <tabstop>_autosave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ExportGraphicsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_autosave</sender>
   <signal>toggled(bool)</signal>
   <receiver>_period</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
